specification/eventgrid/Azure.Messaging.EventGrid/tsp-output/@azure-tools/typespec-autorest/2023-06-01-preview/openapi.json

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
This isn't the correct output location for this json file. Also probably shouldn't be name openapi.json but named after the service instead.